assets/app/view/game/round/stock.rb

```diff
@@ -350,6 +356,68 @@ def render_buy_input_interval(company)
             *buy_buttons,
           ])]
         end
+
+        def render_bidbox(companies)
```

Comment:
I wonder if a special render is needed. Other games include privates in the stock round (1860 and 1824), but for purchasing. It would reduce complexity if we did it the same way here and only added the bid button logic. I suggest taking a look.

Reply:
I check how all other have done it, and also talked with Per who implements 1824. I want to seperate the bidboxes in its own row.

Reply:
And per also wanted to seperate the minor bids from the rest of the privates.

Reply:
if you can do it the generic way, please remove all of this code